### PR TITLE
Adding OpenShift API article

### PR DESCRIPTION
**Pull request by suehighmoor**

### CODE REVIEW

File: articles/openshift/oshift-ref-self-service-api.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+toc_title: OpenShift API Reference Guide
+toc_fullpath: Reference/oshift-ref-self-service-api.md
+toc_mdlink: oshift-ref-self-service-api.md
+---
+
+# UKCloud for OpenShift API Reference Guide
+
+## Overview
+
+In your UKCloud for OpenShift cluster, there are some operations that you need to perform on the underlying OpenStack infrastructure, such as verifying the status of the VMs underpinning the cluster, or rebooting a node. To help you accomplish these operations, OpenShift provides a REST API.
```
Comment (stevemul) [inline articles/openshift/oshift-ref-self-service-api.md]: It'd be good to be clear this isn't an OpenShift API but a UKCloud API that overlays some elements of the OpenShift and OpenStack API's.
```suggestion
In your UKCloud for OpenShift cluster, there are some operations that you need to perform on the underlying OpenStack infrastructure, such as verifying the status of the VMs underpinning the cluster, or rebooting a node. To help you accomplish these operations, UKCloud provide a REST API.
```

**Comment by suehighmoor** [comment]
@stevemul, @stevedicko, what's the status with this?

**Comment by stevemul** [comment]
@suehighmoor - I'm not sure we're going to be releasing this feature now. The new version of OpenShift will make it defunct and the issues around support of it were never fully resolved. Do you want to remove the PR and branch or get confirmation from @stevedicko?

**Comment by suehighmoor** [comment]
Feature no longer being released. Closing pull request and deleting branch.

**Pull request closed without merging**